admin-dev/themes/new-theme/scss/pages/_product_page.scss

```diff
@@ -1065,3 +1065,63 @@
     }
   }
 }
+
+#select2-product_combination_bulk_images-results .select2-results__option--highlighted[aria-selected],
+#select2-product_combination_bulk_images-results .select2-results__option[aria-selected=true] {
+  background-color: inherit;
+  color: inherit;
+}
+
+.combination-bulk-images {
+  width: 2.857rem;
+  height: 2.857rem;
+  background-color: $gray-light;
+  border: solid 0.07143rem $gray-light;
+}
+
+.combination-bulk-images-label {
+  height: 1.429rem;
+  object-fit: contain;
+  font-family: $font-family-base;
+  font-size: 0.9rem;
+  line-height: 1.43;
+  color: $gray-medium;
+  margin: 0.6429rem 0 0 0.6429rem;
+}
+
+#select2-product_combination_bulk_images-results {
+  li.select2-results__option input[type=checkbox] {
+    margin: 0 0.625rem 0 0.313rem;
+    vertical-align: middle;
+  }
+
+  &::-webkit-scrollbar {
+    width: 0.5714rem;
+    background-color: rgba(187, 205, 210, 0.3);
+  }
+
+  &::-webkit-scrollbar-thumb {
+    width: 0.5714rem;
+    border-radius: 7.143rem;
+    background-color: $gray-light;
+  }
+
+}
+
+#select2-product_combination_bulk_images-container .select2-container--prestakit .select2-selection--multiple {
+  padding: inherit;
+  padding-right: 1.429rem;
+
+  &::before {
```

Comment:
Hi @fatmaBouchekoua, I think you could use the new version of UI-Kit without override select2 input style: https://www.npmjs.com/package/prestakit